cluster/run_tests.sh

```diff
@@ -0,0 +1,3 @@
+pushd tests
+go test -master=http://192.168.200.2:8184
```

Comment:
Instad of hardcoding the IP here you can do:

source hack/config.sh
go test -master=http://${master_ip}:8184

Maybe you want to add a $master_port to the config too?